### PR TITLE
[KOGITO-815] - Create image for Task Console

### PR DESCRIPTION
**Pull request by spolti**
Signed-off-by: spolti <fspolti@redhat.com>

See: https://issues.redhat.com/browse/KOGITO-815

Many thanks for submitting your Pull Request :heart:! 

Please make sure your PR meets the following requirements:

- [x] You have read the [contributors guide](README.md#contributing-to-kogito-images-repository)
- [x] Pull Request title is properly formatted: `[KOGITO-XYZ] Subject`
- [x] Pull Request contains link to the JIRA issue
- [x] Pull Request contains description of the issue
- [x] Pull Request does not include fixes for issues other than the main ticket
- [x] Your feature/bug fix has a testcase that verifies it
- [ ] You've tested the new feature/bug fix in an actual OpenShift cluster

### CODE REVIEW

**Comment by spolti** [comment]
@pefernan please take a look and let me know what are the configurations missing.

File: kogito-task-console-overrides.yaml [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+- name: "io.openshift.expose-services"
+  value: "8080:http"
+
+envs:
+  - name: "SCRIPT_DEBUG"
+    example: "true"
+    description: "If set to true, ensures that the bash scripts are executed with the -x option, printing the commands and their arguments as they are executed. Also debug JVM initialization."
+  - name: "KOGITO_DATAINDEX_HTTP_URL"
+    example: "http://data-index.service:8080"
+    description: "Data Index service address, defaults tohttp://localhost:8180."
```
Comment (pefernan) [inline kogito-task-console-overrides.yaml]: white space missing here

Comment (spolti) [reply kogito-task-console-overrides.yaml]: good catch, thanks

**Review by pefernan — APPROVED** [review]
+1 I'd say we are ok to go. 

There's a whitespace missing in the KOGITO_DATAINDEX_HTTP_URL description, but everything else looks fine to me

File: Makefile [hunk 2]
```diff
@@ -145,6 +145,19 @@ ifneq ($(findstring rc,$(IMAGE_VERSION)), rc)
 	${BUILD_ENGINE} tag quay.io/kiegroup/kogito-management-console:${IMAGE_VERSION} quay.io/kiegroup/kogito-management-console:${SHORTENED_LATEST_VERSION}
 endif
 
+# build the quay.io/kiegroup/kogito-task-console image
+kogito-task-console:
+ifneq ($(ignore_build),true)
+	${CEKIT_CMD} build --overrides-file kogito-task-console-overrides.yaml ${BUILD_ENGINE}
+endif
+# if ignore_test is set tu true, ignore the tests
```
Comment (Kaitou786) [inline Makefile]:
```suggestion
# if ignore_test is set to true, ignore the tests
```

**Review by Kaitou786 — APPROVED** [review]

File: Makefile (hunk 2 above)
Comment (spolti) [reply Makefile]: it was on all other images hehe, thanks for finding this.

**Review by radtriste — APPROVED** [review]

**Pull request merged**